<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuoteInput</class>
 <widget class="QWidget" name="QuoteInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>445</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,1,4">
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="quoteLE">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="featuresGB">
     <property name="title">
      <string>features</string>
     </property>
     <layout class="QVBoxLayout" name="featuresLayout"/>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="ownerCB">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>group</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QCheckBox" name="obscenceCheck">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>is obscene</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>text</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QGroupBox" name="attrsGB">
     <property name="title">
      <string>attributes</string>
     </property>
     <layout class="QFormLayout" name="formLayout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
